src/main/java/duke/exceptions/DukeInvalidCommandException.java

```diff
@@ -0,0 +1,26 @@
+package duke.exceptions;
+
+
+/**
+ * Duke Invalid Command Exception class.
```

Comment:
Can write something like "Throws custom exception commands if something goes wrong when processing user input.". I'm not too sure whats the difference between your DukeException and DukeInvalidCommandException, so do change the class description accordingly